modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/manager/DefaultPortalKaleoManager.java

```diff
@@ -321,15 +323,15 @@ private String _getLocalizedTitle(long companyId, String definitionName)
 		return _localization.getXml(localizedValuesMap, "title");
 	}
 
-	private static final String _DEFINITION_NAME = "Single Approver";
```

Comment:
Move this variable to `WorkflowDefinitionConstants` as:
`public static final String NAME_SINGLE_APPROVER = "Single Approver";`